src/huggingface_hub/_login.py

```diff
@@ -128,10 +128,20 @@ def interpreter_login() -> None:
     _|_|_|_|  _|    _|  _|  _|_|  _|  _|_|    _|    _|  _|  _|  _|  _|_|      _|_|_|    _|_|_|_|  _|        _|_|_|
     _|    _|  _|    _|  _|    _|  _|    _|    _|    _|    _|_|  _|    _|      _|        _|    _|  _|        _|
     _|    _|    _|_|      _|_|_|    _|_|_|  _|_|_|  _|      _|    _|_|_|      _|        _|    _|    _|_|_|  _|_|_|_|
-
-    To login, `huggingface_hub` now requires a token generated from https://huggingface.co/settings/tokens .
     """
     )
+    if token is not None:
+        print(
+            "    A token is already saved on your machine. Run `huggingface - cli"
+            " whoami` to get more information or `huggingface - cli logout` if you want"
+            " to log out. Setting a new token will erase the existing one."
+        )
+        return
+
+    print(
+        "    To login, `huggingface_hub` now requires a token generated from"
+        " https://huggingface.co/settings/tokens ."
+    )
```

Comment:
I would prefer if we still allow the user to erase its credentials directly instead of asking for a proper logout/login
i.e. do not `return` if a token has been found. The user can still do Ctrl+c if needed.

```suggestion
    if HfFolder.get_token() is not None:
        print(
            "    A token is already saved on your machine. Run `huggingface-cli"
            " whoami` to get more information or `huggingface-cli logout` if you want"
            " to log out."
        )
        print("    Setting a new token will erase the existing one.")

    print(
        "    To login, `huggingface_hub` requires a token generated from"
        " https://huggingface.co/settings/tokens ."
    )
```